extern/test_bmi_py/bmi_grid.py

```diff
@@ -0,0 +1,200 @@
+"""bmi_grid.py
+Module for supporting BMI grid meta data and functionality
+
+@author Nels Frazier
+@version 0.1
+"""
+
+from enum import Enum
+from functools import reduce
+from typing import TYPE_CHECKING
+import numpy as np
+
+if TYPE_CHECKING:
+    from typing import Tuple
+    from numpy.typing import NDArray
+
+class GridType(str, Enum):
+    """
+        Enumeration of supported BMI grid types (https://bmi.readthedocs.io/en/stable/#get-grid-type)
+    """
+    scalar = "scalar",
```

Comment:
Enum standard seems to be UPPERCASE but I can't find a reference... PEP 8 doesn't specifically address Enums?